app/assets/stylesheets/application/modules/_discussion.scss

```diff
@@ -0,0 +1,337 @@
+  }
+  a {
+    color: unset;
+    text-decoration: unset;
+    &:not(:last-child) > .discussion {
+      border-bottom: 0;
+    }
+  }
+
+  .discussions-reset-query {
+    color: #6a737d;
+    font-weight: 600;
+    &:hover {
+      color: inherit;
+    }
+  }
+
+  .discussion-pagination {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .discussion-language-icon {
+    float: right;
+    margin-right: 10px;
+    margin-top: 6px;
+    -ms-transform: scale(1.4, 1.4);
+    -webkit-transform: scale(1.4, 1.4);
+    transform: scale(1.4, 1.4);
+  }
+
+  .discussion-icon {
+    -ms-transform: scale(0.8, 0.8);
+    -webkit-transform: scale(0.8, 0.8);
+    transform: scale(0.8, 0.8);
+    margin-left: 5px;
+    float: right;
+    .fa-stack-1x {
+      -ms-transform: scale(0.9, 0.9);
+      -webkit-transform: scale(0.9, 0.9);
+      transform: scale(0.9, 0.9);
+    }
+  }
+}
+
+.discussions-no-filtered-results {
+  padding: 20px;
+  border: 1px solid #e1e4e8;
+  text-align: center;
+  div {
+    margin-bottom: 20px;
+    i {
+      font-size: 28px;
+    }
+  }
+  span {
+    font-size: 24px;
+  }
+}
+
+.discussion-status-icon {
+  padding-right: 10px;
+}
+
+.discussion-row {
+  display: table-cell;
+  vertical-align: middle;
+  .discussion-title {
+    font-size: 18px;
+    font-weight: bold;
+  }
+  .discussion-description {
+    overflow: hidden;
+    display: block;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+}
+
+.discussion-body {
+  display: none;
+}
+
+.discussion-display-more {
+  display: table-cell;
+  width: 30px;
+  margin-right: 0;
+  a {
+    cursor: pointer;
+  }
+}
+
+.discussion-messages-icon {
+  margin-left: 30px;
+  margin-bottom: 5px;
+}
+
+.discussions-toolbar {
+  width: 100%;
+  padding: 10px 20px;
+  display: table;
+  margin-top: 20px;
+  background-color: #f6f8fa;
+  border: 1px solid #e1e4e8;
+  border-radius: 3px 3px 0 0;
+  border-bottom: none;
+  .discussions-toolbar-status {
+    float: left;
+    a {
+      margin-right: 15px;
+      &:not(.selected) {
+        .status-icon, span {
+          color: #808080;
+        }
+      }
+      &:hover {
+        color: darken(#808080, 20%);
+      }
+      &.selected {
+        span {
+          color: darken(#808080, 40%);
+          font-weight: bold;
+        }
+      }
+    }
+  }
+  .discussions-toolbar-filter {
+    float: right;
+    margin-right: 20px;
+    cursor: pointer;
+    li.selected {
+      a {
+        font-weight: bold;
+        padding-left: 2px;
+        &::before {
+          content: '✓';
+        }
+      }
+    }
+  }
+}
+
+.message-divider {
+  border-top: 1px solid #ecf0f1;
+  margin-top: 20px
+}
+
+.discussion-user-menu {
+  a {
+    padding: 5px 10px;
+    font-size: 15px;
+    line-height: 20px;
+    border: 1px solid rgba(27, 31, 35, 0.2);
+    border-radius: 0.25em;
+    background-color: #fafafa;
+    font-weight: bold;
+    color: #000000;
```

Comment:
Can we do something to avoid hardcoding colors? Can we extract variables? @fedescarpa please share your thoughts here.